python/requirements.txt

```diff
@@ -0,0 +1,11 @@
+blinker==1.7.0
+cffi==1.16.0
+click==8.1.7
+cryptography==41.0.7
+Flask==3.0.0
```

Comment:
I don't think you need any of the dependencies here except Flask / PyJWT. Remove all and try creating the venv again and it should work. They do look like dependencies for Flask, which will be pulled in automatically. I guess you are pinning them to ensure they work? Not sure if that's necessary / it makes it looks like your simple app requires and uses all these things and might confuse users as they would likely copy/paste this long list into their app. For this demo, it might be better to just keep it simple and pin Flask and PyJWT only (what you directly use in the app.py) and remove everything else.